pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -84,6 +84,13 @@ func (e *InvalidRegistries) Error() string {
 	return e.s
 }
 
+// ValidatePrefix calls parseURL but discards the normalized registry value returned
```

Comment:
(Absolutely non-blocking: This is an implementation detail, the public callers don’t care.)